groups/alteryx-sandbox/nlb.tf

```diff
@@ -0,0 +1,99 @@
+# Load Balancer
```

Comment:
Remove the comment here as it generally doesn't provide any additional context over that provided by the resource itself:

```suggestion
```